challenges/classes.js

```diff
@@ -1,7 +1,40 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+// function CuboidMaker(attributes){
+//     this.length = attributes.length;
+//     this.width = attributes.width;
+//     this.height = attributes.height
+//   }
 
+  class CuboidMaker {
+      constructor(attr){
+          this.length = attr.length;
+          this.width = attr.width;
+          this.height = attr.height;
+      }
+      volume() {
+        return this.length * this.width * this.height;
+      }
+      surfaceArea() {
+        return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+      }
+  }
+
+//   CuboidMaker.prototype.volume = function() {
+//     return this.length * this.width * this.height;
+//   }
+
+//   CuboidMaker.prototype.surfaceArea = function() {
+//     return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+//   }
+
+  const cuboid = new CuboidMaker({
+    length: 4,
+    width: 5,
+    height: 5
```

Comment:
The `new`keyword is correctly implemented to create new object of CuboidMaker type.